sphinx-docs/screenshot.py

```diff
@@ -24,7 +24,7 @@
 OUTPUT_PATH = os.path.realpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), SS_DUMP_DIR))
 MANAGE_PATH = os.path.realpath(os.path.join(os.path.dirname(os.path.abspath(__file__)),"..","kalite","manage.py"))
 # Formatted from subprocess.Popen
-SCREENSHOT_COMMAND = ["/usr/bin/python", MANAGE_PATH, "screenshots"]
+SCREENSHOT_COMMAND = ["C:\\Python27\\python.exe", MANAGE_PATH, "screenshots"]
```

Comment:
This should be:
`SCREENSHOT_COMMAND = [sys.executable, MANAGE_PATH, "screenshots"]`